### PR TITLE
Email changed to Username

### LINKED ISSUE #71: username instead of email
### Description

In the sign in field the email field be username and not email 
since in the cause of signing up there is no field requesting for an email

an image of this issue can be seen below
Screenshot from 2022-09-03 12-13-33

### PR DESCRIPTION
**Pull request by mofirojean**
### Contributor Checklist

Fixes #71

- [ ] Did you add comments for every public function/method?
- [x] Did you specify the type of improvement?
- [x] Did you request a desired merge strategy?

#### Type of Improvement

- Better user experience

#### Backend Code Generation Impact

- 

#### Desired Merge Strategy

- Squash

#### Release Notes

-

### Reviewer Checklist (only modified by reviewer)
- [x] Did you add the appropriate labels?
- [x] Did you review?
- [x] Did you check whether all relevant Contributor checkboxes have been checked?

### CODE REVIEW

File: src/app/modules/rc-auth/components/login/login.component.html [hunk 1]
```diff
@@ -6,7 +6,7 @@
 </div>
 
 <form [formGroup]="loginForm" (submit)="loginAction()">
-  <label for="email" class="block text-900 font-medium mb-2">Email</label>
+  <label for="email" class="block text-900 font-medium mb-2">Username</label>
```
Comment (elroykanye) [inline src/app/modules/rc-auth/components/login/login.component.html:9]: This will be better as `Email/Username`

**Review by elroykanye — COMMENTED** [review]
Thanks

**Review by elroykanye — CHANGES_REQUESTED** [review]

File: src/app/modules/rc-auth/components/login/login.component.html (hunk 1 above)
[the remark this answers is not in the document]
Comment (mofirojean) [reply src/app/modules/rc-auth/components/login/login.component.html]: I just corrected it

**Comment by mofirojean** [comment]
Hey @elroykanye 
i have changed the name as you required 

but what about the formControlName in the code below
<input id="email" type="text" pInputText class="w-full mb-3" [formControlName]="'username'"

**Comment by elroykanye** [comment]
It is not an issue and does not relate to this fix. Let us keep fixes specific, as anything changing about form control name with break the functionality of the form and consequent request body sent to the backend.

**Comment by mofirojean** [comment]
Ok Thanks for the clarification

**Comment by mofirojean** [comment]
waiting for you to merge

File: src/app/modules/rc-auth/components/login/login.component.html (hunk 1 above)
Comment (mofirojean) [reply src/app/modules/rc-auth/components/login/login.component.html]: Ok let me get to it

**Review by elroykanye — APPROVED** [review]

**Pull request merged**